CMakeLists.txt

```diff
@@ -149,6 +149,11 @@ install(TARGETS libvast_internal EXPORT VASTTargets)
 # Require standard C++17
 target_compile_features(libvast_internal INTERFACE cxx_std_17)
 
+# GNU prior to 9.1 requires linking with -lstdc++fs when using std::filesystem library
+if (CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND CMAKE_CXX_COMPILER_VERSION STRLESS "9.1.0")
```

Comment:
This should probably use `VERSION_LESS` rather than `STRLESS`.

Reply:
Also, thinking about this some more, this should probably link publicly against the `libvast` target rather than the `libvast_internal` target.

Reply:
Thanks for reminding me about `VERSION_LESS`. Just switched to that and also applied the linking flag to `libvast` rather than `libvast_internal`. 👍